aws/websites/metrics.pytorch.org/files/dashboards/squid_proxy_status.json

```diff
@@ -339,7 +339,7 @@
           "refId": "Unhealthy",
           "region": "default",
           "statistics": [
-            "Sum"
+            "Average"
```

Comment:
sum is not correct to capture what's the number of unhealthy instances across 30s time window